### ISSUE TITLE
Comparison of Sherpa with other similar products available in the market

### ISSUE THREAD

**Issue by babasahebpinjar** [Aggregate-Intellect/sherpa#266]
Introduction

Overview of  Sherpa, key features and capabilities that makes it stand out.

We can use a template as below

Competitor 
Overview
Product Name: 
Description:
Key Features
    [List of key features]
    [Any unique capabilities]
Strengths
   [Competitor's strengths compared to Sherpa]
Weaknesses
**Sherpa's capabilities over all the above features**

**Comment by amirfz** [comment]
I'm a bit confused. are you suggesting to add info like this to the docs?
if so, I haven't seen any projects do that. have you?

**Comment by babasahebpinjar** [comment]
Yes, a document would be great which does the comparison of sherpa with other products in the market.

I haven't come across anything in any other projects, but such document will help someone who wants to make a decision why go forward with sherpa instead of others. Also, keeping a comparison document and updating it on a regular basis will give us insights what features to be included next.

**Comment by amirfz** [comment]
yeah, I haven't seen other projects do that. i wonder how they clarify why
they should be used and all that
